src/core/Elsa.Core/ElsaConfigurationsOptions.cs

```diff
@@ -17,9 +17,9 @@
 
 namespace Elsa
 {
-    public class ElsaOptions
+    public class ElsaConfigurationsOptions
```

Comment:
Would `ElsaConfiguration` be a better name? Combining "configuration" and "options" seems a bit much.